app/views/transaction/show.html.erb

```diff
@@ -24,6 +24,8 @@
   edition: @edition
 } do %>
   <section class="intro">
+    <%= render 'transaction/country_notice' if @publication.show_country_notice? %>
```

Comment:
I feel like there's perhaps not enough about this code that screams this is a temporary experiment. I'd wonder if it's simple to just put all the code in this view so it's all compartmentalised - especially since the HTML is short.

Something like:

```suggestion
    <%# Temporary experiment to test if applicability banner increases devolved administration usage %>
    <% if @publication.base_path == "/business-coronavirus-support-finder" && Date.today < Date.new(2021, 4, 17) %>
        <div class="country-notice">
          <p class="country-notice__text">
            Applies to: <strong>England, Scotland, Wales and Northern&nbsp;Ireland</strong>
          </p>
        </div>
    <% end %>
```

Reply:
good point - I'll add your suggestion in

Reply:
I wanted to keep the stuff in the presenter to keep it a bit cleaner, I'll add the comment in though